app/controllers/v0/sign_in_controller.rb

```diff
@@ -232,12 +232,14 @@ def logout_get_redirect_url
     end
 
     def handle_pre_login_error(error, client_id)
-      if SignIn::Constants::ClientConfig::COOKIE_AUTH.include?(client_id)
+      if client_config(client_id).valid? && client_config(client_id).cookie_auth?
```

Comment:
I don't think we need `client_config(client_id).valid?`, that part's implicit in the creation of `client_config` (just invoking the initializer will fail if it's not valid)